modules/ossm-directing-egress-traffic-through-a-gateway-using-istio-apis.adoc

```diff
@@ -0,0 +1,226 @@
+// This procedure is used in the following assembly:
+// * gateways/ossm-directing-outbound-traffic-through-a-gateway
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-directing-egress-traffic-through-a-gateway-using-istio-apis_{context}"]
+= Directing egress traffic through a gateway using Istio APIs
+:context: ossm-directing-egress-traffic-through-a-gateway-using-istio-apis
```

Comment:
Same comment about the context that I don't think should be set here

Reply:
done